samples/OpenIdConnectSample/Startup.cs

```diff
@@ -102,7 +98,6 @@ await context.Authentication.SignOutAsync(OpenIdConnectDefaults.AuthenticationSc
                     {
                         RedirectUri = "/signedout"
                     });
-                    await context.Authentication.SignOutAsync(CookieAuthenticationDefaults.AuthenticationScheme);
```

Comment:
In user code. SignInScheme isn't very accessible here. Also keeping in sync with templates: 
https://github.com/aspnet/Templates/blob/dev/src/Rules/StarterWeb/OrganizationalAuth/Single/Common/Controllers/AccountController.cs#L24